### PR TITLE
Fix broken post edit form page

### PR DESCRIPTION
**Pull request by taniki**

### CODE REVIEW

File: js/components/post/form.vue [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
 <script>
 import Post from 'models/post';
-import VerticalForm from 'components/form/vertical-form.vue';
 
 export default {
     name: 'post-form',
-    components: {VerticalForm},
+    components: {
+            vform: require('components/form/vertical-form.vue')
```
Comment (noirbizarre) [inline js/components/post/form.vue]: As much as possible we avoid inline require

File: js/components/post/form.vue [hunk 2]
```diff
@@ -1,16 +1,17 @@
 <template>
 <div>
-<vertical-form v-ref:form :fields="fields" :model="post"></vertical-form>
+    <vform v-ref:form :fields="fields" :model="post"></vform>
```
Comment (noirbizarre) [inline js/components/post/form.vue]: We are doing the opposite, ie. transform `vfrom`/`hform` into  `vertical-form` `horizontal-form` for the following reason:
- this is more explicit
- web component spec prohibe single term markup (you need a dash in a web component markup)

Comment (taniki) [reply js/components/post/form.vue]: argh. I was overzealous at getting a homogeneous code from the wrong example.

**Comment by taniki** [comment]
@noirbizarre revert done

**Comment by noirbizarre** [comment]
:ok_hand:

**Pull request merged**